requirements.txt

```diff
@@ -1,5 +1,7 @@
 checklist==0.0.10
 spacy==2.2.4
+pandas>=1.2.0
```

Comment:
We recommend creating a separate requirements.txt in the same transformation folder so that the main requirements.txt is not changed often (which might change the version of the dependency for specific transformation)

Reply:
Ok noted, I have moved this requirement into the transformation folder.